Runtime/Scripts/RGServiceManager.cs

```diff
@@ -172,7 +173,11 @@ public async Task GetExternalConnectionInformationForBotInstance(long botInstanc
                         RGDebug.LogDebug($"RG Bot Instance external connection info: {connInfo}");
                         onSuccess.Invoke(connInfo);
                     },
-                    onFailure: async (f) => { onFailure.Invoke(); }
+                    onFailure: async (f) =>
+                    {
+                        RGDebug.LogWarning($"Failed retrieving external connection information: {f}");
```

Comment:
This one we definitely do not want to log here.  This API polls the backend waiting for the bot to start and will repeatedly throw an http error code or timeout while the bot is still starting up.  I had a log like this in here originally and it fills up the log with spam on every bot startup as part of our normal operation.

Reply:
Agreed, I removed it